addons/website_quote/__init__.py

```diff
@@ -3,3 +3,14 @@
 
 from . import controllers
 from . import models
+
+from odoo.api import Environment, SUPERUSER_ID
+
+def _install_sale_payment(cr):
+    env = Environment(cr, SUPERUSER_ID, {})
+    sale_payment = env['ir.module.module'].search([
+        ('name', '=', 'sale_payment'),
+    ], limit=1)
```

Comment:
The limit is superfluous and you could just add `('state', '=', 'uninstalled')` instead of doing the if below, but this is just me nitpicking.

Reply:
It does make the code a bit cleaner. Doing it this way might call `button_install` on an empty recordset causing messages like:

```
ALLOW access to module.button_install on [] to user admin #1 via 127.0.0.1
```

but I suppose that's not a big deal.